scripts/generate-app-config/generateSettingsConfig.js

```diff
@@ -12,8 +12,8 @@ const generateUtilitiesConfig = (appConfig) => {
     themes: appConfig.themes || {},
     defaultLocale: appConfig.defaultLocale,
     locales: appConfig.locales || {},
-    defaultDirection: appConfig.defaultDir,
-    directions: ['ltr', 'rtl'],
+    defaultDirection: appConfig.defaultDirection,
+    directions: appConfig.bidirectional ? ['ltr', 'rtl'] : [],
```

Comment:
@emilyrohrbough I functionally tested it but should there be a _wdio_ test? I might need a separate site config just for that, then?

Reply:
@nramamurth you should be able to you write a jest test to verify the generated config

Reply:
Ah, my bad. Here it is -  https://github.com/cerner/terra-dev-site/pull/210/commits/9a33b01a68dc5fe84a94ecbb237dd34430713c49